ui-core/src/main/java/com/adyen/checkout/components/ui/util/PdfOpener.kt

```diff
@@ -15,11 +15,10 @@ import android.net.Uri
 import android.os.Build
 import androidx.browser.customtabs.CustomTabColorSchemeParams
 import androidx.browser.customtabs.CustomTabsIntent
-import com.adyen.checkout.components.ui.util.ThemeUtil
 import com.adyen.checkout.core.log.LogUtil
 import com.adyen.checkout.core.log.Logger
 
-internal class PdfOpener {
+class PdfOpener {
```

Comment:
We had a discussion about that actually, any class that need to launch custom tabs needs to rely on `ThemeUtil` which is currently a class in `ui-core` because it tries to fetch the primary theme color. You could argue that `PdfOpener` (and even `RedirectHandler`) are not UI related classes but then what about `ThemeUtil`? 🤔

Reply:
Joseph and I discussed that yesterday and I wanted to put it actually inside `components-core` but I would depend on `ui-core` and that would make a circular dependency between the two modules as we need to use `ThemeUtil` from `ui-core` so I put it in `ui-core` as there's something related to the ui which is `ThemeUtil` to get primary color for the custom tabs. 
PS: If you have other suggestions please let me know.

Reply:
There are multiple solutions here:
1. Move `PdfOpener` to `online-banking-core`. For the time being only online banking is using it. We'll have to find a solution for these dependencies once we have to move it
2. Move `ThemeUtil` with `PdfOpener` to `components-core` or `checkout-core`. Doesn't really solve anything, because now `ThemeUtil` is in a wrong module
3. Ideally, `ui-core` shouldn't have a dependency on `components-core`. It should be the other way around imo. This will require a lot of work because of `AdyenLinearLayout`

Option 1 is best for now I think, but we should go for 3 on the long term if everyone agrees

Reply:
will move it to `online-banking-core` for now as you suggested and eventually will go with your 3rd option